resources/ext/config/conf.d/puppetserver.conf

```diff
@@ -74,3 +74,9 @@ profiler: {
     # enable or disable profiling for the Ruby code; defaults to 'true'.
     #enabled: true
 }
+
+# settings related to the certificate authority
+certificate-authority: {
+    # allow CA to sign certificate requests that have subject alternative names.
+    # allow-subject-alt-names: false
```

Comment:
Should I leave this commented out?

Reply:
Yeah, having the defaults commented out at least matches the rest of the file